build.sh

```diff
@@ -1,8 +1,27 @@
 #!/usr/bin/env bash
+set -e
 
 if hash powershell 2>/dev/null; then
     echo 'Continuing with `powershell -noprofile -c Start-PSBuild`'
     powershell -noprofile -c "Import-Module ./build.psm1; Start-PSBuild"
 else
-    echo 'No `powershell`, see docs/building/linux.md or macos.md to build PowerShell!'
+   echo 'Continuing with full manual build'
```

Comment:
Maybe a separate script (i.e. `build-new-platform.sh` or something) would be better. The reason is the build system is already pretty complicated. Putting more logic into the main build path would increase the probability of future errors and time for trouble-shooting.